python/nano/src/bigdl/nano/deps/onnxruntime/tensorflow/tensorflow_onnxruntime_model.py

```diff
@@ -89,8 +100,6 @@ def on_forward_start(self, inputs):
         return inputs
 
     def on_forward_end(self, outputs):
-        if isinstance(outputs, list) and len(outputs) == 1:
-            outputs = outputs[0]
         outputs = self.numpy_to_tensors(outputs)
```

Comment:
if output is a multilayer embedded list， does `self.numpy_to_tensors(outputs)` still works?

Reply:
yes, it works, <https://github.com/intel-analytics/BigDL/blob/79f1df8f472fa858e30ade8a1992cb4a965250d7/python/nano/src/bigdl/nano/utils/inference/tf/model.py#L73>